### ISSUE TITLE
`poetry install` exits after summarizing the execution plan

### ISSUE THREAD

**Issue by FranzForstmayr** [python-poetry/poetry#7148]
- **Poetry version**: 1.2.2
- **Python version**: 3.9.15
- **OS version and name**: Ubuntu 20.04.5 LTS
- **pyproject.toml**: 

```toml
[tool.poetry]
name = "test_app"
version = "1.1.0"
description = "Test Application"
authors = [
    "Name Surname <name@mail.com>",
]
license = "proprietary"

[tool.poetry.dependencies]
python = "~3.9"
openpyxl = "^3.0.9"
pandas = "^1.4.1"
dash-extensions = "^0.1.6"
scipy = "^1.9.3"

[tool.poetry.dev-dependencies]
flake8 = "^4.0.1"
flake8-bugbear = "^22.1.11"
flake8-docstrings = "^1.6.0"
pep8-naming = "^0.12.1"
flake8-formatter-junit-xml = "^0.0.6"
pytest = "^7.1.0"
nox = "^2022.1.7"
flake8-annotations = "^2.7.0"
Sphinx = "^4.4.0"
sphinx-rtd-theme = "^1.0.0"
sphinx-autodoc-typehints = "^1.17.0"
black = "^22.1.0"
pytest-cov = "^3.0.0"

[build-system]
requires = ["poetry-core>=1.0.0"]
build-backend = "poetry.core.masonry.api"
```

- [x] I am on the [latest](https://github.com/python-poetry/poetry/releases/latest) stable Poetry version, installed using a recommended method.
- [x] I have searched the [issues](https://github.com/python-poetry/poetry/issues) of this repo and believe that this is not a duplicate.
- [x] I have consulted the [FAQ](https://python-poetry.org/docs/faq/) and [blog](https://python-poetry.org/blog/) for any relevant entries or release notes.
- [x] If an exception occurs when executing a command, I executed it again in debug mode (`-vvv` option) and have included the output below.

## Issue
When I try to install this project on this certain machine, nothing happens.
Verbose option does not show anything useful. Is there another way to debug this issue?

```
(test-app-py3.9) host@HOST:~/apps/test-app$ poetry install -vvv
Loading configuration file /home/host/.config/pypoetry/config.toml
Loading configuration file /home/host/.config/pypoetry/auth.toml
Using virtualenv: /home/host/apps/test-app/.venv
Project environment contains an empty path in sys_path, ignoring.
Installing dependencies from lock file

Finding the necessary packages for the current system

Package operations: 81 installs, 0 updates, 0 removals
```

**Comment by FranzForstmayr** [comment]
I activated the env manually, none of the specified packages was installed, only the default ones:
```
Package    Version
---------- -------
pip        22.3
setuptools 65.5.0
wheel      0.37.1

[notice] A new release of pip available: 22.3 -> 22.3.1
[notice] To update, run: pip install --upgrade pip
```

No, there's nothing below `package operations`, I posted the full output.
It's strange, because I can trigger this only on this machine for now, however I don't know how to debug.

**Comment by FranzForstmayr** [comment]
Reinstalling poetry did not solve the issue.

**Comment by clementdavies56** [comment]
I ran into the same issue , clear the cache in the environment and run it again ,(including default cache).

**Comment by Davidnet** [comment]
I ran also into the same issue, I did the same as above, from what I gather in doing `-vvvv`

Something is wrong in the connectionpool:

```
Package operations: 119 installs, 0 updates, 0 removals, 1 skipped

  • Installing attrs (22.1.0): Pending...
  • Installing platformdirs (2.5.4): Pending...
  • Installing pyrsistent (0.19.2): Pending...
  • Installing six (1.16.0): Pending...
  • Installing traitlets (5.6.0): Pending...
[keyring.backend] Loading KWallet
[keyring.backend] Loading KWallet
[keyring.backend] Loading KWallet
[keyring.backend] Loading KWallet
[keyring.backend] Loading KWallet
[keyring.backend] Loading SecretService
[keyring.backend] Loading SecretService
[keyring.backend] Loading SecretService
[keyring.backend] Loading SecretService
[keyring.backend] Loading SecretService
[keyring.backend] Loading Windows
[keyring.backend] Loading Windows
[keyring.backend] Loading Windows
[keyring.backend] Loading Windows
[keyring.backend] Loading Windows
[keyring.backend] Loading chainer
[keyring.backend] Loading chainer
[keyring.backend] Loading chainer
[keyring.backend] Loading libsecret
[keyring.backend] Loading chainer
[keyring.backend] Loading libsecret
[keyring.backend] Loading chainer
[keyring.backend] Loading libsecret
[keyring.backend] Loading libsecret
[keyring.backend] Loading libsecret
[keyring.backend] Loading macOS
[keyring.backend] Loading macOS
[keyring.backend] Loading macOS
[keyring.backend] Loading macOS
[keyring.backend] Loading macOS
Creating new session for pypi.org
[urllib3.connectionpool] Starting new HTTPS connection (1): pypi.org:443
[urllib3.connectionpool] Starting new HTTPS connection (2): pypi.org:443
[urllib3.connectionpool] Starting new HTTPS connection (3): pypi.org:443
[urllib3.connectionpool] Starting new HTTPS connection (4): pypi.org:443
[urllib3.connectionpool] Starting new HTTPS connection (5): pypi.org:443
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/traitlets/5.6.0/json HTTP/1.1" 304 0
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/platformdirs/2.5.4/json HTTP/1.1" 200 3503
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/six/1.16.0/json HTTP/1.1" 304 0
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/attrs/22.1.0/json HTTP/1.1" 304 0
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/pyrsistent/0.19.2/json HTTP/1.1" 304 0
```

It keeps there for a while, if I do curl to those endpoints, it looks like fine, but do not know why poetry hangs

**Comment by sjperkins** [comment]
I run into similar issues to https://github.com/python-poetry/poetry/issues/7148#issuecomment-1341453253.

Here is an example of this failing with `poetry self update` on `version 1.2.1`

```bash
$ poetry self update -vvv
Loading configuration file /home/simon/.config/pypoetry/config.toml
Updating Poetry version ...

[keyring.backend] Loading KWallet
[keyring.backend] Loading SecretService
[keyring.backend] Loading Windows
[keyring.backend] Loading chainer
[keyring.backend] Loading libsecret
[keyring.backend] Loading macOS
No suitable keyring backend found
No suitable keyring backends were found
Keyring is not available, credentials will be stored and retrieved from configuration files as plaintext.
Creating new session for pypi.org
[urllib3.connectionpool] Starting new HTTPS connection (1): pypi.org:443
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/poetry/json HTTP/1.1" 304 0
```

**Comment by sjperkins** [comment]
And on poetry 1.2.2.

```bash
$ poetry self update -vvv
Loading configuration file /home/simon/.config/pypoetry/config.toml
Updating Poetry version ...

[keyring.backend] Loading KWallet
[keyring.backend] Loading SecretService
[keyring.backend] Loading Windows
[keyring.backend] Loading chainer
[keyring.backend] Loading libsecret
[keyring.backend] Loading macOS
No suitable keyring backend found
No suitable keyring backends were found
Keyring is not available, credentials will be stored and retrieved from configuration files as plaintext.
Creating new session for pypi.org
Source (PyPI): 83 packages found for poetry *
Using version ^1.2.2 for poetry

Updating dependencies
Resolving dependencies...
   1: fact: poetry-instance is 1.2.2
   1: derived: poetry-instance
   1: fact: poetry-instance depends on poetry (^1.2.2)
   1: selecting poetry-instance (1.2.2)
   1: derived: poetry (>=1.2.2,<2.0.0)
Source (PyPI): 1 packages found for poetry >=1.2.2,<2.0.0
   1: fact: poetry (1.2.2) depends on poetry-core (1.3.2)
   1: fact: poetry (1.2.2) depends on poetry-plugin-export (>=1.1.2,<2.0.0)
   1: fact: poetry (1.2.2) depends on cachecontrol (>=0.12.9,<0.13.0)
   1: fact: poetry (1.2.2) depends on cachy (>=0.3.0,<0.4.0)
   1: fact: poetry (1.2.2) depends on cleo (>=1.0.0a5,<2.0.0)
   1: fact: poetry (1.2.2) depends on crashtest (>=0.3.0,<0.4.0)
   1: fact: poetry (1.2.2) depends on html5lib (>=1.0,<2.0)
   1: fact: poetry (1.2.2) depends on importlib-metadata (>=4.4,<5.0)
   1: fact: poetry (1.2.2) depends on jsonschema (>=4.10.0,<5.0.0)
   1: fact: poetry (1.2.2) depends on keyring (>=21.2.0)
   1: fact: poetry (1.2.2) depends on packaging (>=20.4)
   1: fact: poetry (1.2.2) depends on pexpect (>=4.7.0,<5.0.0)
   1: fact: poetry (1.2.2) depends on pkginfo (>=1.5,<2.0)
   1: fact: poetry (1.2.2) depends on platformdirs (>=2.5.2,<3.0.0)
   1: fact: poetry (1.2.2) depends on requests (>=2.18,<3.0)
   1: fact: poetry (1.2.2) depends on requests-toolbelt (>=0.9.1,<0.10.0)
   1: fact: poetry (1.2.2) depends on shellingham (>=1.5,<2.0)
   1: fact: poetry (1.2.2) depends on tomlkit (>=0.11.1,<0.11.2 || >0.11.2,<0.11.3 || >0.11.3,<1.0.0)
   1: fact: poetry (1.2.2) depends on virtualenv (>=20.4.3,<20.4.5 || >20.4.5,<20.4.6 || >20.4.6)
   1: fact: poetry (1.2.2) depends on xattr (>=0.9.7,<0.10.0)
   1: fact: poetry (1.2.2) depends on urllib3 (>=1.26.0,<2.0.0)
   1: fact: poetry (1.2.2) depends on dulwich (>=0.20.46,<0.21.0)
   1: selecting poetry (1.2.2)
   1: derived: dulwich (>=0.20.46,<0.21.0)
   1: derived: urllib3 (>=1.26.0,<2.0.0)
   1: derived: xattr (>=0.9.7,<0.10.0)
   1: derived: virtualenv (>=20.4.3,!=20.4.5,!=20.4.6)
   1: derived: tomlkit (>=0.11.1,!=0.11.2,!=0.11.3,<1.0.0)
   1: derived: shellingham (>=1.5,<2.0)
   1: derived: requests-toolbelt (>=0.9.1,<0.10.0)
   1: derived: requests (>=2.18,<3.0)
   1: derived: platformdirs (>=2.5.2,<3.0.0)
   1: derived: pkginfo (>=1.5,<2.0)
   1: derived: pexpect (>=4.7.0,<5.0.0)
   1: derived: packaging (>=20.4)
   1: derived: keyring (>=21.2.0)
   1: derived: jsonschema (>=4.10.0,<5.0.0)
   1: derived: importlib-metadata (>=4.4,<5.0)
   1: derived: html5lib (>=1.0,<2.0)
   1: derived: crashtest (>=0.3.0,<0.4.0)
   1: derived: cleo (>=1.0.0a5,<2.0.0)
   1: derived: cachy (>=0.3.0,<0.4.0)
   1: derived: cachecontrol[filecache] (>=0.12.9,<0.13.0)
   1: derived: poetry-plugin-export (>=1.1.2,<2.0.0)
   1: derived: poetry-core (==1.3.2)
   1: fact: dulwich (0.20.50) depends on urllib3 (>=1.25)
   1: selecting dulwich (0.20.50)
   1: selecting urllib3 (1.26.12)
   1: fact: virtualenv (20.16.7) depends on distlib (>=0.3.6,<1)
   1: fact: virtualenv (20.16.7) depends on filelock (>=3.4.1,<4)
   1: fact: virtualenv (20.16.7) depends on platformdirs (>=2.4,<3)
   1: selecting virtualenv (20.16.7)
   1: derived: filelock (>=3.4.1,<4)
   1: derived: distlib (>=0.3.6,<1)
   1: selecting tomlkit (0.11.6)
   1: selecting shellingham (1.5.0)
   1: fact: requests-toolbelt (0.9.1) depends on requests (>=2.0.1,<3.0.0)
   1: selecting requests-toolbelt (0.9.1)
   1: fact: requests (2.28.1) depends on charset-normalizer (>=2,<3)
   1: fact: requests (2.28.1) depends on idna (>=2.5,<4)
   1: fact: requests (2.28.1) depends on urllib3 (>=1.21.1,<1.27)
   1: fact: requests (2.28.1) depends on certifi (>=2017.4.17)
   1: selecting requests (2.28.1)
   1: derived: certifi (>=2017.4.17)
   1: derived: idna (>=2.5,<4)
   1: derived: charset-normalizer (>=2,<3)
   1: selecting platformdirs (2.5.4)
   1: selecting pkginfo (1.8.3)
   1: fact: pexpect (4.8.0) depends on ptyprocess (>=0.5)
   1: selecting pexpect (4.8.0)
   1: derived: ptyprocess (>=0.5)
   1: fact: packaging (21.3) depends on pyparsing (>=2.0.2,<3.0.5 || >3.0.5)
   1: selecting packaging (21.3)
   1: derived: pyparsing (>=2.0.2,!=3.0.5)
   1: fact: keyring (23.11.0) depends on jaraco.classes (*)
   1: fact: keyring (23.11.0) depends on importlib-metadata (>=4.11.4)
   1: fact: keyring (23.11.0) depends on SecretStorage (>=3.2)
   1: fact: keyring (23.11.0) depends on jeepney (>=0.4.2)
   1: fact: keyring (23.11.0) depends on pywin32-ctypes (<0.1.0 || >0.1.0,<0.1.1 || >0.1.1)
   1: selecting keyring (23.11.0)
   1: derived: pywin32-ctypes (!=0.1.0,!=0.1.1)
   1: derived: jeepney (>=0.4.2)
   1: derived: SecretStorage (>=3.2)
   1: derived: importlib-metadata (>=4.11.4)
   1: derived: jaraco.classes
   1: fact: jsonschema (4.17.0) depends on attrs (>=17.4.0)
   1: fact: jsonschema (4.17.0) depends on importlib-resources (>=1.4.0)
   1: fact: jsonschema (4.17.0) depends on pkgutil-resolve-name (>=1.3.10)
   1: fact: jsonschema (4.17.0) depends on pyrsistent (>=0.14.0,<0.17.0 || >0.17.0,<0.17.1 || >0.17.1,<0.17.2 || >0.17.2)
   1: selecting jsonschema (4.17.0)
   1: derived: pyrsistent (>=0.14.0,!=0.17.0,!=0.17.1,!=0.17.2)
   1: derived: pkgutil-resolve-name (>=1.3.10)
   1: derived: importlib-resources (>=1.4.0)
   1: derived: attrs (>=17.4.0)
   1: fact: html5lib (1.1) depends on six (>=1.9)
   1: fact: html5lib (1.1) depends on webencodings (*)
   1: selecting html5lib (1.1)
   1: derived: webencodings
   1: derived: six (>=1.9)
   1: selecting crashtest (0.3.1)
   1: fact: cleo (1.0.0a5) depends on pylev (>=1.3.0,<2.0.0)
   1: fact: cleo (1.0.0a5) depends on crashtest (>=0.3.1,<0.4.0)
   1: selecting cleo (1.0.0a5)
   1: derived: pylev (>=1.3.0,<2.0.0)
   1: selecting cachy (0.3.0)
   1: fact: cachecontrol (0.12.11) depends on cachecontrol (0.12.11)
   1: fact: cachecontrol (0.12.11) depends on requests (*)
   1: fact: cachecontrol (0.12.11) depends on msgpack (>=0.5.2)
   1: fact: cachecontrol (0.12.11) depends on lockfile (>=0.9)
   1: selecting cachecontrol[filecache] (0.12.11)
   1: derived: lockfile (>=0.9)
   1: derived: msgpack (>=0.5.2)
   1: derived: cachecontrol (==0.12.11)
   1: fact: poetry-plugin-export (1.2.0) depends on poetry (>=1.2.2,<2.0.0)
   1: fact: poetry-plugin-export (1.2.0) depends on poetry-core (>=1.3.0,<2.0.0)
   1: selecting poetry-plugin-export (1.2.0)
   1: selecting poetry-core (1.3.2)
   1: selecting filelock (3.8.0)
   1: selecting distlib (0.3.6)
   1: selecting certifi (2022.9.24)
   1: selecting idna (3.4)
   1: selecting charset-normalizer (2.1.1)
   1: selecting ptyprocess (0.7.0)
   1: selecting pyparsing (3.0.9)
   1: fact: jaraco-classes (3.2.3) depends on more-itertools (*)
   1: selecting jaraco-classes (3.2.3)
   1: derived: more-itertools
   1: selecting pyrsistent (0.19.2)
   1: selecting attrs (22.1.0)
   1: selecting webencodings (0.5.1)
   1: selecting six (1.16.0)
   1: selecting pylev (1.4.0)
   1: selecting msgpack (1.0.4)
   1: selecting more-itertools (9.0.0)
   1: fact: xattr (0.9.9) depends on cffi (>=1.0)
   1: selecting xattr (0.9.9)
   1: derived: cffi (>=1.0)
   1: fact: cffi (1.15.1) depends on pycparser (*)
   1: selecting cffi (1.15.1)
   1: derived: pycparser
   1: selecting pycparser (2.21)
   1: fact: importlib-metadata (4.13.0) depends on zipp (>=0.5)
   1: selecting importlib-metadata (4.13.0)
   1: derived: zipp (>=0.5)
   1: selecting zipp (3.10.0)
   1: selecting pywin32-ctypes (0.2.0)
   1: selecting jeepney (0.8.0)
   1: fact: secretstorage (3.3.3) depends on cryptography (>=2.0)
   1: fact: secretstorage (3.3.3) depends on jeepney (>=0.6)
   1: selecting secretstorage (3.3.3)
   1: derived: cryptography (>=2.0)
   1: fact: cryptography (38.0.3) depends on cffi (>=1.12)
   1: selecting cryptography (38.0.3)
   1: selecting pkgutil-resolve-name (1.3.10)
   1: fact: importlib-resources (5.10.0) depends on zipp (>=3.1.0)
   1: selecting importlib-resources (5.10.0)
   1: selecting lockfile (0.12.2)
   1: fact: cachecontrol (0.12.11) depends on requests (*)
   1: fact: cachecontrol (0.12.11) depends on msgpack (>=0.5.2)
   1: selecting cachecontrol (0.12.11)
   1: Version solving took 0.175 seconds.
   1: Tried 1 solutions.

Finding the necessary packages for the current system
Source (poetry-repo): 1 packages found for poetry >=1.2.2,<2.0.0

Package operations: 1 install, 11 updates, 0 removals, 34 skipped

  • Updating certifi (2022.12.7 -> 2022.9.24): Pending...
[urllib3.connectionpool] Starting new HTTPS connection (1): pypi.org:443
  • Updating cryptography (38.0.4 -> 38.0.3): Pending...
[urllib3.connectionpool] Starting new HTTPS connection (2): pypi.org:443
  • Updating urllib3 (1.26.13 -> 1.26.12): Pending...
[urllib3.connectionpool] Starting new HTTPS connection (3): pypi.org:443
  • Updating zipp (3.11.0 -> 3.10.0): Pending...
[urllib3.connectionpool] Starting new HTTPS connection (4): pypi.org:443
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/certifi/2022.9.24/json HTTP/1.1" 200 2271
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/urllib3/1.26.12/json HTTP/1.1" 200 17300
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/zipp/3.10.0/json HTTP/1.1" 200 2046
[urllib3.connectionpool] https://pypi.org:443 "GET /pypi/cryptography/38.0.3/json HTTP/1.1" 200 5967
```

**Comment by FranzForstmayr** [comment]
The last output on my machine is `Package Operations`, so probably this is something different.

**Comment by sjperkins** [comment]
> The last output on my machine is Package Operations, so probably this is something different.

Ah thanks for clarifying.

> I ran into the same issue , clear the cache in the environment and run it again ,(including default cache).

This fixed my issue.

**Comment by FranzForstmayr** [comment]
Clearing cache does not work.

```
host@Host:~/apps/test-app$ poetry cache list
PyPI
_default_cache
gitlab
pypi
host@Host:~/apps/test-app$ poetry cache clear _default_cache --all
Delete 0 entries? (yes/no) [yes]
host@Host:~/apps/test-app$ poetry cache clear PyPI --all
Delete 0 entries? (yes/no) [yes]
host@Host:~/apps/test-app$ poetry cache clear gitlab --all
Delete 0 entries? (yes/no) [yes]
host@Host:~/apps/test-app$ poetry cache clear pypi --all
Delete 0 entries? (yes/no) [yes]
host@Host:~/apps/test-app$ poetry install
Loading configuration file /home/host/.config/pypoetry/config.toml
Loading configuration file /home/host/.config/pypoetry/auth.toml
Using virtualenv: /home/host/apps/test-app/.venv
Project environment contains an empty path in sys_path, ignoring.
Installing dependencies from lock file

Finding the necessary packages for the current system

Package operations: 100 installs, 0 updates, 0 removals
```

**Comment by clementdavies56** [comment]
> Clearing cache does not work.
> 
> ```
> host@Host:~/apps/test-app$ poetry cache list
> PyPI
> _default_cache
> gitlab
> pypi
> host@Host:~/apps/test-app$ poetry cache clear _default_cache --all
> Delete 0 entries? (yes/no) [yes]
> host@Host:~/apps/test-app$ poetry cache clear PyPI --all
> Delete 0 entries? (yes/no) [yes]
> host@Host:~/apps/test-app$ poetry cache clear gitlab --all
> Delete 0 entries? (yes/no) [yes]
> host@Host:~/apps/test-app$ poetry cache clear pypi --all
> Delete 0 entries? (yes/no) [yes]
> host@Host:~/apps/test-app$ poetry install
> Loading configuration file /home/host/.config/pypoetry/config.toml
> Loading configuration file /home/host/.config/pypoetry/auth.toml
> Using virtualenv: /home/host/apps/test-app/.venv
> Project environment contains an empty path in sys_path, ignoring.
> Installing dependencies from lock file
> 
> Finding the necessary packages for the current system
> 
> Package operations: 100 installs, 0 updates, 0 removals
> ```

1) Clear cache 
2)Restart Terminal
3) Run poetry install again

**Comment by Davidnet** [comment]
> I run into similar issues to [#7148 (comment)](https://github.com/python-poetry/poetry/issues/7148#issuecomment-1341453253).
> 
> Here is an example of this failing with `poetry self update` on `version 1.2.1`
> 
> ```shell
> $ poetry self update -vvv
> Loading configuration file /home/simon/.config/pypoetry/config.toml
> Updating Poetry version ...
> 
> [keyring.backend] Loading KWallet
> [keyring.backend] Loading SecretService
> [keyring.backend] Loading Windows
> [keyring.backend] Loading chainer
> [keyring.backend] Loading libsecret
> [keyring.backend] Loading macOS
> No suitable keyring backend found
> No suitable keyring backends were found
> Keyring is not available, credentials will be stored and retrieved from configuration files as plaintext.
> Creating new session for pypi.org
> [urllib3.connectionpool] Starting new HTTPS connection (1): pypi.org:443
> [urllib3.connectionpool] https://pypi.org:443 "GET /pypi/poetry/json HTTP/1.1" 304 0
> ```

Hey I solved by blowing up all the caches, as you said I really think something is wrong with connection pool

**Comment by georgeseifada** [comment]
I also faced the same issue in the latest versions of poetry. It could be related to https://github.com/python-poetry/poetry/issues/6459 because this issue was [not present in `1.1.14` but it does come up in `>=1.2.0`](https://github.com/python-poetry/poetry/issues/6459#issuecomment-1347057585)

**Comment by FranzForstmayr** [comment]
does not help in this case.

**Comment by neersighted** [comment]
I'd like to clarify that this issue is very specific and relates to no output/Poetry silently exiting after planning its install operation. The other comments here are off-topic and relate to different issues entirely, many of which are solved in 1.3.1.

@FranzForstmayr, can you please test on 1.3.1. Additionally, can you please provide the output of `stty size`? I suspect this may be a duplicate of #7184, or at the very least the same code is swallowing an exception.

**Comment by FranzForstmayr** [comment]
stty size returns "58 209"

I upgraded to 1.3.1 in the meantime, but still same issue. But it seems not affected from project files (pyproject.toml, poetry.lock)..

**Comment by neersighted** [comment]
Can you also provide the exit code? `poetry install; echo $?` should be sufficient.

**Comment by FranzForstmayr** [comment]
1

**Comment by neersighted** [comment]
Okay, looks like another specialized variant of #7184; the root cause is different but the effect (swallowing an exception) is the same. Can you run Poetry with `--no-ansi` and see what output you get?

**Comment by jaoxford** [comment]
Same thing happens for us,

When running `poetry install -vvv; echo $?` from CodeFresh. We get the following:
```python
Loading configuration file /codefresh/volume/yo-main-application/poetry.toml
Using virtualenv: /root/.venv
Installing dependencies from lock file

Finding the necessary packages for the current system

Package operations: 211 installs, 0 updates, 0 removals

1
```

1 being the exit code.

**Comment by neersighted** [comment]
> Can you run Poetry with `-vvv --no-ansi` and see what output you get?

**Comment by jaoxford** [comment]
> > Can you run Poetry with `-vvv --no-ansi` and see what output you get?

I followed the recommendations in here: https://github.com/python-poetry/poetry/issues/7184 to use `--no-ansi` and it solved our issue.

**Comment by FranzForstmayr** [comment]
Finally more output:

```
Loading configuration file /home/host/.config/pypoetry/config.toml
Loading configuration file /home/host/.config/pypoetry/auth.toml
Using virtualenv: /home/rd-app-host/apps/test-app/.venv
Installing dependencies from lock file

Finding the necessary packages for the current system

Package operations: 41 installs, 0 updates, 0 removals, 62 skipped


  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:251 in _execute_operation
      249|                 if self._should_write_operation(operation):
      250|                     if not operation.skipped:
    > 251|                         self._io.write_line(

  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:251 in _execute_operation
      249|                 if self._should_write_operation(operation):
      250|                     if not operation.skipped:
    > 251|                         self._io.write_line(

  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:251 in _execute_operation
      249|                 if self._should_write_operation(operation):
      250|                     if not operation.skipped:
    > 251|                         self._io.write_line(

  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:251 in _execute_operation
      249|                 if self._should_write_operation(operation):
      250|                     if not operation.skipped:
    > 251|                         self._io.write_line(

  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:251 in _execute_operation
      249|                 if self._should_write_operation(operation):
      250|                     if not operation.skipped:
    > 251|                         self._io.write_line(

  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:251 in _execute_operation
      249|                 if self._should_write_operation(operation):
      250|                     if not operation.skipped:
    > 251|                         self._io.write_line(

  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:251 in _execute_operation
      249|                 if self._should_write_operation(operation):
      250|                     if not operation.skipped:
    > 251|                         self._io.write_line(
```

sorry for the delay, was not in the office for a few days.

**Comment by onnodb** [comment]
Encountering the same issue. Perhaps the following information helps in troubleshooting?

Poetry version: 1.3.2
Python version: 3.10.6
OS: Xubuntu 22.04.1 LTS

Poetry was installed with the official installer on a plain-vanilla Xubuntu installation (in a VM).

`poetry -vvv install` runs all the way until the output:
```
Writing lock file

Finding the necessary packages for the current system:
Source (poetry-repo): 1 packages found for mypy >= ...
(list of other packages)
```
and then exits with exit code 1. No actual installation of any packages is performed, and the environment remains empty (as checked with `pip list` inside the env).

The lack of any error messages or feedback is rather baffling.

Running `poetry -vvv --no-ansi install` instead gives the same unhandled exception and stack trace [as reported above by FranzForstmayr](https://github.com/python-poetry/poetry/issues/7148#issuecomment-1375288691).

Same issue occurs both with the config setting `virtualenvs.in-project` set to `true`, as well as its default `false`. `stty size` returns `55 271` (running in the default xfce4-terminal).

**Comment by FranzForstmayr** [comment]
@neersighted 

I finally upgraded to 1.3.2 but still had the same issue.
While digging around a little bit, I finally found a workaround.

I removed the bullet characters in https://github.com/python-poetry/poetry/blob/master/src/poetry/installation/executor.py#L245 and the following lines and poetry works again on this machine.

**Comment by FranzForstmayr** [comment]
as it seems to be terminal related, maybe this information also helps.

This is a virtual ubuntu machine without graphical interface. I usually connect from a windows computer with powershell,the new windows terminal application or VS Code remote terminal.

**Comment by jcrawfordor** [comment]
Experiencing the same problem with Poetry 1.4.2 on Debian 11.6 with Python 3.9.

Output of "poetry install -vvv --no-ansi":
```
Loading configuration file /root/.config/pypoetry/config.toml
Using virtualenv: <snip>
Installing dependencies from lock file

Finding the necessary packages for the current system

Package operations: 25 installs, 0 updates, 0 removals


  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:268 in _execute_operation
      266|                 if self._should_write_operation(operation):
      267|                     if not operation.skipped:
    > 268|                         self._io.write_line(

  Stack trace:

  4  ~/.local/share/pypoetry/venv/lib/python3.9/site-packages/poetry/installation/executor.py:268 in _execute_operation
      266|                 if self._should_write_operation(operation):
      267|                     if not operation.skipped:
    > 268|                         self._io.write_line(
```

Without --no-ansi there is no output after printing the plan summary. This is occurring when accessing the machine via SSH with or without tmux.

**Comment by sadahry** [comment]
`--no-ansi` option works for me on circleci job.

**Comment by JDJGInc** [comment]
I got this issue on this machine:

```bash
neofetch                                                                                                                                                                                         [14:14:36]
       _,met$$$$$gg.          jdjg@raspberrypi 
    ,g$$$$$$$$$$$$$$$P.       ---------------- 
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 11 (bullseye) aarch64 
 ,$$P'              `$$$.     Host: Raspberry Pi 4 Model B Rev 1.1 
',$$P       ,ggs.     `$$b:   Kernel: 6.1.31-v8+ 
`d$$'     ,$P"'   .    $$$    Uptime: 32 days, 5 hours, 3 mins 
 $$P      d$'     ,    $$P    Packages: 1491 (dpkg) 
 $$:      $$.   -    ,d$$'    Shell: zsh 5.8 
 $$;      Y$b._   _,d$P'      Resolution: 1920x1080 
 Y$$.    `.`"Y$$$$P"'         DE: LXDE 
 `$$b      "-.__              Theme: PiXflat [GTK3] 
  `Y$$                        Icons: PiXflat [GTK3] 
   `Y$$.                      Terminal: lxterminal 
     `$$b.                    Terminal Font: Monospace 10 
       `Y$$b.                 CPU: BCM2835 (4) @ 1.500GHz 
          `"Y$b._             Memory: 519MiB / 1849MiB 
              `"""
                                                      
                                                      

```

**Comment by JDJGInc** [comment]
https://mystb.in/BeginningPathsConvertible

The error.

**Comment by JDJGInc** [comment]
Grabbed with ``poetry install -vvv > food.txt ``

**Comment by dimbleby** [comment]
should have been fixed by cleo 2.1, https://github.com/python-poetry/cleo/pull/299
